sats-dna/src/main/kotlin/com/sats/dna/components/schedule/Schedule.kt

```diff
@@ -134,7 +145,7 @@ private fun Preview() {
         ScheduledWorkout(
             id = "foo",
             day = "Today",
-            time = "09:00",
+            time = "09:00 PM",
```

Comment:
Since we love nitpicking: this is not a likely time format. When using AM/PM, we wouldn't use a leading zero in front of the hour, so it'd be “9:00 AM”. :nerd_face: 

(I think it's also common to ditch the minutes for the “00” minutes, but that doesn't happen with java.time formatting, AFAICT.)